pkg/deployment/reconcile/plan_builder_tls.go

```diff
@@ -419,6 +460,9 @@ func createKeyfileRenewalPlanMode(
 
 func checkServerValidCertRequest(ctx context.Context, context PlanBuilderContext, apiObject k8sutil.APIObject, group api.ServerGroup, member api.MemberStatus, ca resources.Certificates) (*tls.ConnectionState, error) {
 	endpoint := fmt.Sprintf("https://%s:%d", k8sutil.CreatePodDNSNameWithDomain(apiObject, context.GetSpec().ClusterDomain, group.AsRole(), member.ID), shared.ArangoPort)
+	if group == api.ServerGroupSyncMasters {
+		endpoint = fmt.Sprintf("https://%s:%d/%s", k8sutil.CreatePodDNSNameWithDomain(apiObject, context.GetSpec().ClusterDomain, group.AsRole(), member.ID), shared.ArangoSyncMasterPort, "_api/version")
```

Comment:
Maybe move "_api/version" to constants.go similarly to we have `ArangoExporterStatusEndpoint` const